src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/probationoffendersearch/Address.kt

```diff
@@ -3,7 +3,27 @@ package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.probationoffende
 import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Address as IntegrationAPIAddress
 
 data class Address(
+  val addressNumber: String?,
+  val district: String?,
+  val buildingName: String?,
+  val county: String?,
+  val from: String?,
   val postcode: String?,
+  val streetName: String?,
+  val to: String?,
+  val town: String?,
```

Comment:
Can we reorder them?
  val addressNumber: String?,
  val buildingName: String?,
  val streetName: String?,
  val town: String?,
  val district: String?,
  val county: String?,
  val postcode: String?,
  val from: String?,
  val to: String?,